### PR TITLE
feat: add nuxt-hue

### PR DESCRIPTION
**Pull request by lihbr**
A great addition to the Nuxt module ecosystem™

[Caramelldansen_Cat_Extended](https://www.youtube.com/watch?v=7K7kx1HDp-Y)

Let me know if anything ☺

### CODE REVIEW

File: modules/nuxt-hue.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+github: https://github.com/lihbr/nuxt-hue
+website: https://github.com/lihbr/nuxt-hue
+learn_more: ''
+category: Devtools
+type: 3rd-party
+maintainers:
+  - name: lihbr
+    github: lihbr
+compatibility:
+  nuxt: ^2.0.0
```
Comment (pi0) [inline modules/nuxt-hue.yml:15]: Isn't nuxt3 supported? 

```suggestion
  nuxt: ^2.0.0 || ^3.0.0
```

File: modules/nuxt-hue.yml [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+  requires: {}
```
Comment (pi0) [inline modules/nuxt-hue.yml:16]:
```suggestion
  requires: { bridge: 'optional' }
```

File: modules/nuxt-hue.yml (hunk 1 above)
Comment (lihbr) [reply modules/nuxt-hue.yml]: It is!

File: modules/nuxt-hue.yml (hunk 2 above)
Comment (lihbr) [reply modules/nuxt-hue.yml]: 🚀

**Comment by lihbr** [comment]
Alright, all good ☺️

**Review by antfu — APPROVED** [review]

**Pull request merged**